src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonTypeInfoOfT.WriteHelpers.cs

```diff
@@ -76,7 +77,7 @@ public partial class JsonTypeInfo<T>
                 Debug.Assert(CanUseSerializeHandler);
                 Debug.Assert(Converter is JsonMetadataServicesConverter<T>);
 
-                using var bufferWriter = new PooledByteBufferWriter(Options.DefaultBufferSize);
```

Comment:
Naming conventions in runtime state that `var` is permitted if the type name is clearly visible on the RHS (i.e. via a `new` expression).